roles/ceph-rgw/tasks/docker/rgw_pool_pgs_docker.yml

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
keep the task `set_fact` rename the fact to `docker_exec_rgw_cmd` and move it in `roles/ceph-rgw/tasks/main.yml`, drop the rest (remaining task + file)

Reply:
Isn't it better to keep the docker/containerized stuff all together inside tasks/docker instead of having the code spread all over the role?

Actually I was going to propose to move this fact inside tasks/docker/main.yml:
https://github.com/ceph/ceph-ansible/blob/v3.0.27/roles/ceph-rgw/tasks/main.yml#L2

Reply:
@jtudelag in fact, not really.
We aim to get rid of these kind of splits because this duplicates lot of code while it's not necessary. So let's write patches in that sense when it's possible (the `{{ docker_exec_cmd }}` facts were even created for that.).
By the way, this approach would fix the non-containerized case, which at first sight, seems to be broken.